package.json

```diff
@@ -25,23 +25,25 @@
     "bin"
   ],
   "scripts": {
-    "lint": "eslint \"./bin/*.js\" \"./test/**/*.js\" \"{packages}/**/!(node_modules)/*.js\" ",
+    "appveyor:lint": "lerna bootstrap && npm run lint",
+    "appveyor:test": "npm run test",
+    "bootstrap": "npm run clean:all && npm install && lerna bootstrap",
+    "build": "tsc",
+    "bundlesize": "bundlesize",
+    "changelog": "conventional-changelog --config ./build/changelog-generator/index.js --infile CHANGELOG.md --same-file",
+    "clean:all": "rimraf node_modules package-lock.json packages/*/node_modules packages/*/package-lock.json",
     "format": "prettier-eslint ./bin/*.js ./test/**/*.js ./packages/**/*.js --write",
+    "jsdoc": "jsdoc -c jsdoc.json -r -d docs",
     "lint:codeOnly": "eslint \"{bin}/**/!(__testfixtures__)/*.js\" \"{bin}/**.js\"",
+    "lint": "eslint \"./bin/*.js\" \"./test/**/*.js\" \"{packages}/**/!(node_modules)/*.js\" ",
     "precommit": "lint-staged",
     "pretest": "npm run lint",
-    "test": "nyc jest",
     "reportCoverage": "nyc report --reporter=json && codecov -f coverage/coverage-final.json --disable=gcov",
-    "jsdoc": "jsdoc -c jsdoc.json -r -d docs",
-    "appveyor:lint": "lerna bootstrap && npm run lint",
-    "appveyor:test": "npm run test",
+    "semantic-release": "semantic-release",
```

Comment:
this should be removed now. Bad rebase? SR is removed from master

Reply:
Fixing this. I think `next` branch will need rebasing as well.

Reply:
`git pull origin master`